web/decap-config/collections/06-tasks.yml

```diff
@@ -916,6 +916,202 @@ collections:
                   widget: "markdown",
                   required: true,
                 }
+      - label: "Cigarette License Step 2"
+        name: "cigaretteLicense-step2"
+        file: "content/src/fieldConfig/cigarette-license-step2.json"
+        editor:
+          preview: true
+        fields:
+          - label: Cigarette License Step 2
+            name: cigaretteLicenseStep2
+            collapsed: false
+            widget: object
+            fields:
+              - label: "Licensee Information Header"
+                name: "licenseeInformationHeader"
+                widget: "string"
+                required: true
+
+              - label: "Licensee Information Description"
+                name: "licenseeInformationDescription"
+                widget: "markdown"
+                required: true
+
+              - label: "Business Name Text"
+                name: "businessNameText"
+                widget: "string"
+                required: true
+
+              - label: "Business Name Error Text"
+                name: "businessNameErrorText"
+                widget: "string"
+                required: true
+
+              - label: "Business Address Header"
+                name: "businessAddressHeader"
+                widget: "string"
+                required: true
+
+              - label: "Business Address Description"
+                name: "businessAddressDescription"
+                widget: "markdown"
+                required: true
+
+              - label: "Mailing Address Header"
+                name: "mailingAddressHeader"
+                widget: "string"
+                required: true
+
+              - label: "Mailing Address Description"
+                name: "mailingAddressDescription"
+                widget: "markdown"
+                required: true
+
+              - label: "Contact Information Header"
+                name: "contactInformationHeader"
+                widget: "string"
+                required: true
+
+              - label: "Contact Information Description"
+                name: "contactInformationDescription"
+                widget: "markdown"
+                required: true
+
+              - label: "Mailing Is Same Checkbox"
+                name: "mailingIsSameCheckbox"
+                widget: "string"
+                required: true
+
+              - label: "Fields"
+                name: "fields"
+                widget: "object"
+                fields:
+                  - label: "Mailing Address Line 1"
```

Comment:
Confirmed in the CMS locally that this preview looks how we want it. We are focusing on the new components that pertain specifically to Cigarette License (Mailing Address and Contact fields) as far as being editable in the cigarette license CMS collection.